yb-voyager/cmd/analyzeSchema.go

```diff
@@ -197,6 +198,30 @@ func reportSummary() {
 		reportStruct.Summary.DBObjects = append(reportStruct.Summary.DBObjects, dbObject)
 	}
 }
+// Checks Whether there is a GIN index
+func checkGin(sqlInfoArr []sqlInfo, fpath string) {
+	for _, sqlInfo := range sqlInfoArr {
+		idx := ginRegex.FindStringSubmatch(sqlInfo.stmt)
+		if idx != nil {
+			columnsFromGin :=  strings.Trim(idx[4], `()`)
+			columnList := strings.Split(columnsFromGin, ",")
+			if len(columnList) > 1 {
```

Comment:
Isn't it enough to check for the presence of a comma to conclude that it is an index on multiple columns?